awx_collection/plugins/modules/tower_inventory_source.py

```diff
@@ -128,6 +128,10 @@
         - list of notifications to send on error
       type: list
       elements: str
+    organization:
+      description:
+        - Name of organization for project.
```

Comment:
name of organization for the inventory source's inventory's organization

Also, connect #5193 as that will be resolved by this

Reply:
Oops, bad copypasta